examples/src/app/(demos)/layout.tsx

```diff
@@ -0,0 +1,12 @@
+import CopyLink from '@/components/CopyLink'
+
+export default function RootLayout({ children }: { children: React.ReactNode }) {
+  return (
+    <>
```

Comment:
I noticed that the container for the demos has been pushed down with the new CopyLink component now above it, but it keeps the height of the window, so it adds extra unneeded scroll to every page. I think we can fix this by wrapping the page in a flexbox container. That would just mean replacing this `<>` with `<div className="h-full flex flex-col">`.

(The Tree CRDT demo sometimes still has extra scroll, but that's because it has a fixed 600x600 size for its SVG element. I think we can leave that as is, though.)